File: cloud/Server/Models/Account.cs
// Copyright © Spatial Corporation. All rights reserved.

using Spatial.Persistence;

namespace Spatial.Cloud.Models;

/// <summary>
/// ...
/// </summary>
[Collection("accounts")]
public class Account : Record
{
    /// <summary>
    /// The user's email address.
    /// </summary>
    public string Email { get; set; }
}

File: interface/data/users/accounts/Account.ts
// Copyright © Spatial Corporation. All rights reserved.

import { Schema } from "../..";

export type Account = Schema<{
  /**
   * The user's name.
   */
  name: string;

  /**
   * The user's email address.
   */
  email: string;

  /**
   * The user's avatar.
   */
  avatar?: string;
}>;
